packages/foreman/foreman-proxy/foreman-proxy.spec

```diff
@@ -117,7 +117,12 @@ cp -p -r bin extra lib modules Rakefile Gemfile.in smart_proxy.gemspec bundler.d
 chmod a+x %{buildroot}%{_datadir}/%{name}/bin/smart-proxy
 rm -rf %{buildroot}%{_datadir}/%{name}/*.rb
 
+# Create etc and var dirs
+%{__mkdir_p} %{buildroot}/etc/foreman-proxy
+%{__mkdir_p} %{buildroot}%{_var}/lib/foreman-proxy
```

Comment:
This should already be covered by line 103, yet on my 1.16.0 proxy it doesn't exist.